docs/content/preview/migrate/migrate-steps.md

```diff
@@ -138,6 +137,13 @@ Using [ora2pg](https://ora2pg.darold.net) and [pg_dump](https://www.postgresql.o
 
 The `yb-voyager export schema` command extracts the schema from the source database, converts it into PostgreSQL format (if the source database is Oracle or MySQL), and dumps the SQL DDL files in the `EXPORT_DIR/schema/*` directories.
 
+{{< note title="Renaming index names for MySQL" >}}
+
+YugabyteDB Voyager renames the indexes for MySQL migrations while exporting the schema.
```

Comment:
Suggestion:

YugabyteDB Voyager renames the indexes for MySQL migrations while exporting the schema.
MySQL supports two or more indexes to have the same name within the same database, provided they are for different tables. Similarly to PostgreSQL, YugabyteDB does not support duplicate index names in the same schema. To avoid index name conflicts during export schema, yb-voyager prefixes each index name with the associated table name.